content/en/docs/release-oversight/payload-testing.md

```diff
@@ -18,3 +18,11 @@ For example, if `/payload 4.10 nightly informing` is issued on a PR, the robot w
 
 The jobs triggered by the command are determined by [OpenShift Release Controllers](/docs/getting-started/useful-links/#services).
 The linked page from [payload-tests portal](https://pr-payload-tests.ci.openshift.org/runs/) at the bottom of the comment shows the status of the payload testing and the details of those jobs.
+
+A particular job or set of jobs can be triggered by `/payload-job`, such as
+
+> /payload-job <periodic_ci_openshift_release_some_job> <periodic_ci_openshift_release_another_job>
```

Comment:
No limit in the current implementation.

We could have also several commands from the same comment.
Each occurrence of a job leads to a run of the job.

Do you have concerns like https://issues.redhat.com/browse/DPTP-2608?

Reply:
yes, but only in the case of making an error.  For example, if you said `/payload-job xxxx xxxx 100` instead of 10 by mistake, you'd launch 100 jobs.  And from what I understand, there's no way to stop the jobs once they startup.

I guess if people are careful with the command, we'll be fine.